content/en/api/integrations_aws/code_snippets/aws_update.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Make sure you replace the <DD_API_KEY> and <DD_APP_KEY> key below
+# with the ones for your account
+
+api_key=<DD_API_KEY>
+app_key=<DD_APP_KEY>
+
+# Update an AWS Account in Datadog
+
+curl -X PUT -H "Content-type: application/json" \
+-d '{
+        "account_id": "<AWS_ACCOUNT_ID>",
+        "filter_tags": ["<KEY>:<VALUE>"],
+        "host_tags": ["<KEY>:<VALUE>"],
+        "role_name": "DatadogAWSIntegrationRole",
+        "account_specific_namespace_rules": {
+        	"auto_scaling": false,
+        	"opsworks": false
+        }
+    }'\
+"https://api.datadoghq.com/api/v1/integration/aws?api_key=${api_key}&application_key=${app_key}"
```

Comment:
This endpoint requires query parameters to know which account to update, 
`/api/v1/integration/aws?account_id=xyz&role_name=abc&api_key=1234&app_key=5678`

Reply:
fixed :) thanks for the feedback 👍